Cargo.toml

```diff
@@ -14,6 +14,7 @@ members = [
     "examples/dodge_the_creeps",
     "examples/resource",
     "examples/native_plugin",
+    "examples/client_server",
```

Comment:
Would it be a good idea to name the example `rpc` instead, so it's more specific?